app/Filament/Resources/EmulatorResource.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Filament\Extensions\Resources\Resource;
+use App\Filament\Resources\EmulatorResource\Pages;
+use App\Filament\Resources\EmulatorResource\RelationManagers\SystemsRelationManager;
+use App\Models\Emulator;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Infolists;
+use Filament\Infolists\Infolist;
+use Filament\Pages\Page;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+
+class EmulatorResource extends Resource
+{
+    protected static ?string $model = Emulator::class;
+
+    protected static ?string $navigationIcon = 'fas-floppy-disk';
+
+    protected static ?string $navigationGroup = 'Platform';
+
+    protected static ?int $navigationSort = 2;
+
+    protected static ?string $recordTitleAttribute = 'handle';
+
+    protected static int $globalSearchResultsLimit = 5;
+
+    /**
+     * @param Emulator $record
+     */
+    public static function getGlobalSearchResultTitle(Model $record): string|Htmlable
+    {
+        return $record->name_full;
+    }
+
+    public static function getGlobalSearchResultDetails(Model $record): array
+    {
+        return [
+            'ID' => $record->ID,
+            'Handle' => $record->handle,
+        ];
+    }
+
+    public static function getGloballySearchableAttributes(): array
+    {
+        return ['ID', 'handle'];
+    }
+
+    public static function infolist(Infolist $infolist): Infolist
+    {
+        return $infolist
+            ->columns(1)
+            ->schema([
+                Infolists\Components\Split::make([
+                    Infolists\Components\Section::make('Details')
+                        ->columns(['xl' => 2, '2xl' => 2])
+                        ->schema([
+                            Infolists\Components\Group::make()
+                                ->schema([
+                                    Infolists\Components\TextEntry::make('handle')
```

Comment:
```suggestion
                    Infolists\Components\Section::make('Details')
                        ->schema([
                            Infolists\Components\Group::make()
                                ->columns(['md' => 2, 'xl' => 3, '2xl' => 4])
                                ->schema([
                                    Infolists\Components\TextEntry::make('handle')
```

I was able to get the fields to fit a little better into their container by moving some of these columns around:

<img width="1129" alt="Screenshot 2024-09-07 at 10 21 28 PM" src="https://github.com/user-attachments/assets/782f9bcb-c8f2-4283-847b-0ca0be76be25">

Reply:
I don't like it.
![image](https://github.com/user-attachments/assets/0308bfa4-92e1-4f3e-a49a-9bfbfd55e0a2)

But two columns seems okay (and matches the edit page)
![image](https://github.com/user-attachments/assets/f73ea441-6bc5-41d6-b0d9-34df4516493a)

Reply:
Two columns looks good 👍